site/content/en/docs/Installation/Creating Cluster/minikube.md

```diff
@@ -44,14 +44,14 @@ some workarounds listed below.
 Other operating systems and drivers may work, but at this stage have not been verified to work with UDP connections
 via Agones exposed ports.
 
-**Linux**
+**Linux (x86-64)**
 * kvm2
 
-**Mac**
+**Mac (x86-64)**
 * Docker (default)
 * Hyperkit
 
-**Windows**
+**Windows (x86-64)**
```

Comment:
🤔 do we want to stick to amd64/arm64 ? That's what we're using in a few other places - just to be consistent?

Reply:
Several places, but this is the most prominent:
https://agones.dev/site/docs/installation/

Reply:
Updated from x86-64 -> amd64.